src/model/commands/accounts.rs

```diff
@@ -149,18 +149,18 @@ fn append_role_command_serialization_and_deserialization() {
 /// transactions or queries, storing signatories, personal data and identifiers.
 #[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
 pub struct CreateAccount {
-    pub account_name: String,
-    pub domain_id: String,
+    pub account_id: Id,
+    pub domain_name: String,
```

Comment:
It seems like there is a duplication here. `Id` already contains `domain_name`.

Reply:
Good point. I think we should change schema of ISI here: https://soramitsu.atlassian.net/browse/RH2-6